modules/cms/src/admin/resources/js/cmsadmin.js

```diff
@@ -815,7 +816,16 @@
 			}
 		});
 
-		/* sidebar logic */
+		/* Blockholder size toggler */
+
+        $scope.isBlockholderSmall = HtmlStorage.getValue('sidebarToggleState', false);
```

Comment:
You should not use the same identifier `sidebarToggleState` as i already do, take another one like `blockHolderToggleState` otherwise the main nav would toggle as well.